sig/app/models/authentication_provider_github.rb

```diff
@@ -0,0 +1,6 @@
+
+class AuthenticationProviderGithub < ApplicationRecord
```

Comment:
This content looks like an RBS, but the file name contains `.rb` extension. Should it be a `.rbs` file?

Reply:
oops

Reply:
@4geru Could you fix this?

Reply:
fixed #4